templates/preview/previewApi.go

```diff
@@ -181,9 +181,9 @@ func (a *Api) generateEmail(templateName models.TemplateName, lang string) (stri
 	content := map[string]interface{}{
 		"Key":                      "123456789123456789123456789123456789",
 		"Email":                    "john@diabeloop.com",
-		"EncodedEmail": url.QueryEscape("john@diabeloop.com"),
+		"EncodedEmail":             url.QueryEscape("john@diabeloop.com"),
 		"FullName":                 "John Doe",
-		"CareteamName":             "John Doe",
+		"PatientName":              "John Doe",
```

Comment:
why do you remove `CareteamName` ?

Reply:
CareTeamName was replaced by PatientName in the english strings, but for the other languages both CareteamName and PatientName are used... I can correct this.